pkg/controller/migplan/migplan_controller_test.go

```diff
@@ -532,3 +532,158 @@ func Test(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcileMigPlan_processProposedPVCapacities(t *testing.T) {
+	type args struct {
+		plan     *migapi.MigPlan
+		analytic *migapi.MigAnalytic
+	}
+	tests := []struct {
+		name               string
+		args               args
+		wantConditions     []migapi.Condition
+		dontWantConditions []migapi.Condition
+		// determines whether or not to assert the message of the condition
+		// when enabled, the test case will also assert whether the message in the conditions are equal
+		assertMessage bool
+	}{
+		{
+			name: "When MigAnaylitic has a ExtendedPVAnalysisFailed condition, MigPlan should show that condition",
```

Comment:
Nit: spelling: `MigAnalytic`

Reply:
Also wondering if this type thing is worthy of a unit test, or adding a unit test here may just make the code harder to change later. Is there complex logic involved here that is prone to failure? Seems kind of simple to me.

Reply:
Since I changed the existing code, I was just making sure, it still works. Agree to your point. worth removing :+1:

Reply:
@djwhatle updated both the warning, and the tests. Let me know if the warning text looks OK to you

Reply:
sorry forgot to update the imports.

Reply:
@djwhatle fixed now